syncd/syncd.cpp

```diff
@@ -3202,6 +3209,12 @@ syncd_restart_type_t handleRestartQuery(swss::NotificationConsumer &restartQuery
         return SYNCD_RESTART_TYPE_FAST;
     }
 
+    if (op == "PRE")
```

Comment:
>PRE [](start = 15, length = 3)

PRE may be not that meaningful. How about PRE-WARM?

Reply:
Expanded to PRE-SHUTDOWN